arviz/data/io_dict.py

```diff
@@ -25,8 +26,17 @@ def __init__(
         observed_data=None,
         constant_data=None,
         predictions_constant_data=None,
+        warmup_posterior=None,
+        warmup_posterior_predictive=None,
+        warmup_predictions=None,
+        warmup_log_likelihood=None,
+        warmup_sample_stats=None,
+        save_warmup=None,
         coords=None,
-        dims=None
+        dims=None,
+        pred_dims=None,
+        pred_coords=None,
```

Comment:
@percygautam can you open an issue so we remember to add these to from_pyro and similars?